Pod/Classes/Acknowledgements/AcknowledgementViewController.swift

```diff
@@ -69,7 +69,7 @@ private extension AcknowledgementViewController {
         labelView.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
             labelView.widthAnchor.constraint(equalTo: view.layoutMarginsGuide.widthAnchor),
-            labelView.topAnchor.constraint(equalTo: view.topAnchor),
+            labelView.topAnchor.constraint(equalTo: view.topAnchor, constant: 16.0),
```

Comment:
@ZevEisenberg could we vertically pad this using attributed string attributes rather than layout constraints to make it easier for consumers to change the padding?